packages/hono/src/build.ts

```diff
@@ -0,0 +1,49 @@
+import { type BuildV3 } from '@vercel/build-utils';
+// @ts-expect-error - FIXME: hono-framework build is not exported
```

Comment:
It was a bit finicky to touch these types since they're hand-written an an `index.d.ts` file. Will revisit